Reference/Searching/Examine/pdf-index.md

```diff
@@ -0,0 +1,85 @@
+---
+versionFrom: 8.0.0
+---
+
+# Pdf indexes and multisearchers
+
+If you want to index pdfs and search for them you will need to use the [UmbracoExamine.Pdf extension package](https://github.com/umbraco/UmbracoExamine.PDF).
+
+## Installation
+
+Install with Nuget:
+`Install-Package UmbracoCms.UmbracoExamine.PDF`
+
+You will then have a new Examine index called "PDFIndex" available.
+
+## Multi index searchers
+
+To use the MultiSearcher in Umbraco 8, you can instantiate it when needed like:
+
+```cs
+using(var multiSearcher = new MultiIndexSearcher("MultiSearcher", new IIndex[] {
+    externalIndex,
+    pdfIndex
+}))
+{
+ ...
+};
+```
+
+Or you can register a multi-index searcher with the ExamineManager on startup like:
+
+```cs
+using Examine;
+using Examine.LuceneEngine.Providers;
+using Umbraco.Core.Composing;
+using UmbracoExamine.PDF;
+
+[ComposeAfter(typeof(ExaminePdfComposer))] //this must execute after the ExaminePdfComposer composer
+public class MyComposer : ComponentComposer<MyComponent>, IUserComposer
+{
+}
+
+public class MyComponent : IComponent
+{
+    private readonly IExamineManager _examineManager;
+
+    public MyComponent(IExamineManager examineManager)
+    {
+        _examineManager = examineManager;
+    }
+
+    public void Initialize()
+    {
+        //Get both the external and pdf index
+        if (_examineManager.TryGetIndex(Constants.UmbracoIndexes.ExternalIndexName, out var externalIndex)
+            && _examineManager.TryGetIndex(PdfIndexConstants.PdfIndexName, out var pdfIndex))
+        {
+            //register a multi searcher for both of them
```

Comment:
Again, is "multi searcher" one word or two words?
Other places, you've written MultiSearcher, or multisearcher 😅 

Sorry if I'm just confusing this more than necessary!